common/components/whats-new-banner/template.njk

```diff
@@ -0,0 +1,14 @@
+<div id="whats-new-banner" class="whats-new-banner" data-module="app-whats-new-banner" data-content-date="{{ params.contentDate }}">
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row banner-padding">
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">{{ params.contentTitle }}</h1>
+        <p class="govuk-body">{{ params.contentDate }}: {{ params.contentBannerText }}.<p>
```

Comment:
I reckon in this component we don't need the `content` prefix as it feels superfluous.

```suggestion
        <h1 class="govuk-heading-l">{{ params.title }}</h1>
        <p class="govuk-body">{{ params.date }}: {{ params.body }}.<p>
```